week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,14 +12,29 @@
  */
 
 class ATM {
-    // Add your code here
-
+	constructor() {
+		this.accountBalance = 100;
+		this.deposit = 0;
+		return this.accountBalance;
+	}
+	make_deposit(deposit) {
+		this.deposit = deposit;
+		return this.deposit;
+	}
+	check_balance() {
+		this.accountBalance += this.deposit;
+		return this.accountBalance;
+	}
+	make_withdrawl(withdraw) {
+		this.withdraw = withdraw;
+		return this.withdraw;
+	}
```

Comment:
Like above, some extra validation on the input value would be welcomed. For example if it is number and you have enough balance on your account (balance >= withdraw).